templates/dials/dial.html

```diff
@@ -0,0 +1,123 @@
+<!DOCTYPE html>
+<html lang="he">
+<head>
+    <meta charset="utf-8">
+    <meta http-equiv="Content-Language" content="he">
+    <title>כמה הכנסת פתוחה?</title>
+</head>
+<body>
+<a href="{% url 'dial-desc' slug %}" target="_blank">
+<svg version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 1097 479" enable-background="new 0 0 1097 479" xml:space="preserve">
+<g id="Layer_2">

```

Comment:
SRSLY? A completely manual svg? no framework to generate it? no higher abstraction? something that no one could ever change or fix?